contrib/docker-compose.yaml

```diff
@@ -51,12 +48,16 @@ services:
         "--rpcpass=[CREATE_A_VERY_HARD_PASSWORD]",
       ]
 
-  bitcoin-lightclient:
-    container_name: bitcoin-lightclient
-    hostname: bitcoin-lightclient
-    build: https://github.com/gonative-cc/bitcoin-lightclient.git#master
+  sui-node:
+    container_name: sui-node
+    image: mysten/sui-tools:devnet
+    volumes:
+      - ./sui_config:/root/.sui/sui_config
```

Comment:
We don't use this volume, right?

Reply:
we do. Its the predefined env and address